### PR TITLE
Max area island DFS problem

### PR DESCRIPTION
**Pull request by sri-vaishnav**
Please complete the following checklist for your PR to be approved. Mark the items by adding an `x`.   
Example:  - [ ] Mark me as completed  
To mark: - [x] Mark me as completed

- [x] My Pull Request adheres to the **Contribution guidelines**.
- [x] I have added `problem.md` and `solution` files in the **appropriate folder**.
- [x] In `problem.md` I have provided a **correct link** to the problem.  
  (In case you are not adding a problem, you can freely mark this as complete)
- [x] The solution I have given is my **own** and not plagiarized in any form.
- [x] My solution passes all the testcases within the given constraints.
- [x] I have properly indented and **documented my solution using comments**.
- [x] *I agree that all the above marked are true to the best of my knowledge and my PR can be closed if found otherwise.*

If any of the above is unmarked, please make the necessary changes and then mark them so that you're PR can be approved and merged.  
(Only completed checklist PRs will be reviewed)   

Hey, it's a popular DFS problem. Can you merge it?

### CODE REVIEW

File: LeetCode_problems/Max Area of Island/solution.cpp [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        
+        else
+          {
+          // Else we make the point visited and then search if its neighbours are 0 or 1 if they are 1 then the size of island is increased, else it is made 0;
+              v[x][y] = true;
+              
+              // we do DFS here
+
+               int    top = dfs(x-1,y,grid,visited);
+
```
Comment (Vishruth-S) [inline LeetCode_problems/Max Area of Island/solution.cpp:20]: Also remove so many unnecessary blank lines and whitespaces in your solution

**Comment by sri-vaishnav** [comment]
Hey vishruth yes i will make those changes.

**Comment by sri-vaishnav** [comment]
Hey, I made those changes. Can you merge please.

**Comment by Vishruth-S** [comment]
Please complete your PR checklist

**Comment by sri-vaishnav** [comment]
Yes completeed.

**Review by Vishruth-S — APPROVED** [review]

**Comment by sri-vaishnav** [comment]
Thank you Vishrith, I will contribute more.

**Pull request merged**